src/ResourceManagement/SqlManagement/Sql.Tests/DatabaseCrudScenarioTests.cs

```diff
@@ -0,0 +1,226 @@
+﻿using Microsoft.Azure.Management.Resources;
+using Microsoft.Azure.Management.Sql;
+using Microsoft.Azure.Management.Sql.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Sql.Tests
+{
+    public class DatabaseCrudScenarioTests
+    {
+        [Fact]
+        public void TestCreateDropDatabase()
+        {
+            string testPrefix = "sqlcrudtest-";
+            string testName = this.GetType().FullName;
+            SqlManagementTestUtilities.RunTestInNewV12Server(testName, "TestCreateDropDatabase", testPrefix, (resClient, sqlClient, resourceGroup, server) =>
+            {
+                Dictionary<string, string> tags = new Dictionary<string, string>()
+                    {
+                        { "tagKey1", "TagValue1" }
+                    };
+
+                // Create database only required parameters
```

Comment:
Discussed offline - because of the huge runtime and because these tests can be considered more E2E than unit tests, leaving like this for now.